.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,25 @@
-### 🔗 Issue Link
-_Jira or Github issue link, if applicable._
+### 🔗 Issue Links
+_Provide all Jira tickets and/or Github issues related to this PR, if applicable._
 
 ### 🎯 Goal
 
 _Describe why we are making this change._
 
 ### 🛠 Implementation
 
-_Provide a description of the implementation._
+_Provide a description of the implementation and list the most important changes._
 
-### 📝 Changes
```

Comment:
Let's keep this in the template. We're leveraging this section when doing a quick summary about changes in PRs that are not carrying the UI changes in them.

Reply:
I find this a bit redundant and most people just remove this section. And if your implementation can be listed in bullet points, you can do it in the "Implementation section", I even updated the description: **Provide a description of the implementation and list the most important changes** 

But I'm fine with reverting these changes 👍 Thoughts @GetStream/ios-developers  ?

Reply:
just ccing one more time -> @GetStream/ios-developers

Reply:
Yes, there's a lot of intersection between changes & implementation sections. I would either go with just `📝 Changes` (as it works for pure documentation PRs) or have both but make `🛠 Implementation` optional.

Reply:
@evsaev @bielikb Can you please re-review? Thanks!